infrastructure/src/main/java/org/wso2/carbon/testgrid/infrastructure/InfrastructureProviderFactory.java

```diff
@@ -32,23 +32,25 @@
 
     static ServiceLoader<InfrastructureProvider> providers = ServiceLoader.load(InfrastructureProvider.class);
 
-    public static InfrastructureProvider getInfrastructureProvider(Infrastructure.ProviderType providerType) throws
+    public static InfrastructureProvider getInfrastructureProvider(Infrastructure infrastructure) throws
```

Comment:
Same as the above comment

Reply:
Fixed with f243d73cb375a67c4fb173c6c2d75a931d9411ae.